functions/requirements.txt

```diff
@@ -0,0 +1,52 @@
+cachecontrol==0.12.12; python_version >= "3.6"
```

Comment:
I've just added a README within the `functions` folder describing this temporary process.
Pretty much cloud functions need a requirements.txt file for gcp to know what to do with them, and in the meantime I thought it could just be manually generated until this part is integrated into the pipeline before terraform runs.